queries/futures/useGetFuturesMarketPositionHistory.ts

```diff
@@ -11,18 +11,18 @@ import { PositionHistory } from './types';
 import { getFuturesEndpoint, mapTradeHistory } from './utils';
 import { getDisplayAsset } from 'utils/futures';
 import { FUTURES_POSITION_FRAGMENT } from './constants';
+import { currentMarketState } from 'store/futures';
 
-const useGetFuturesMarketPositionHistory = (
-	currencyKey: string | null,
-	options?: UseQueryOptions<any | null>
-) => {
+const useGetFuturesMarketPositionHistory = (options?: UseQueryOptions<any | null>) => {
```

Comment:
This query didn't follow the same pattern as you included for the others with setting recoil state and seems to only be used in RefetchContext where the value isn't being used. So not sure it's a redundant query, unless I'm missing something.

Reply:
Nice catch. Pushed a commit that removes the reference to this file. Not sure why we aren't using it anymore, but there's no need to keep calling it.